health/health.d/tcp_resets.conf

```diff
@@ -48,5 +51,7 @@
     warn: $this > ((($1m_ipv4_tcp_resets_received < 5)?(5):($1m_ipv4_tcp_resets_received)) * (($status >= $WARNING)  ? (1) : (10)))
    delay: up 20s down 60m multiplier 1.2 max 2h
  options: no-clear-notification
-    info: average TCP RESETS this host is receiving, over the last 10 seconds (this can be an indication that a service this host needs, has crashed; clear notification for this alarm will not be sent)
+    info: average number of received TCP RESETS over the last 10 seconds \
+          (this can be an indication that a service this host needs, has crashed; \
+          clear notification for this alarm will not be sent)
```

Comment:
```suggestion
    info: average number of received TCP RESETS over the last 10 seconds. \
          This can be an indication that a service this host needs has crashed. \
          Netdata will not send a clear notification for this alarm.
```